### PR TITLE
fix fullscreen button and storage button click blank invalid

### PR DESCRIPTION
**Pull request by devcui**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/cipchk/ng-alain/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

tim 20180412095049
如图：点击全屏按钮或清除缓存按钮空白处无法触发动作

Issue Number: N/A
null

## What is the new behavior?
点击全屏按钮和清除缓存按钮的空白处可以触发动作

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by cipchk** [comment]
i don‘t understand what you mean.

**Comment by devcui** [comment]
改成中文了

**Comment by cipchk** [comment]
我依然还是没明白具体的原因。

**Comment by cipchk** [comment]
补说明：

ziv3aks l axgc2 i o4io

点击红色区域无法触发动作。

File: src/app/layout/default/header/components/fullscreen.component.ts [hunk 1]
```diff
@@ -4,8 +4,10 @@ import * as screenfull from 'screenfull';
 @Component({
     selector: 'header-fullscreen',
     template: `
+    <div>
```
Comment (cipchk) [inline src/app/layout/default/header/components/fullscreen.component.ts:7]: 建议使用
```
styles: [ `host: { display: block }` ]
```

替代。

**Comment by ddpmz** [comment]
<header-fullscreen **class="d-block"**></header-fullscreen>

**Pull request closed without merging**